docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,296 @@
+If not, then now is the time to fix it. Potential fixes include using different versions of a dependency or getting rid of dependencies that we did not end up using.
+
+We actually need to add a dependency ourselves to deploy our backend.
+
+``` xml
+<plugin>
+    <!-- Maven's plugin for copying our dependencies below into our executable Jar file. -->
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-dependency-plugin</artifactId>
+    <version>3.0.1</version>
+    <executions>
+      <execution>
+        <id>copy-dependencies</id>
+        <phase>package</phase>
+        <goals><goal>copy-dependencies</goal></goals>
+      </execution>
+    </executions>
+  </plugin>
+```
+
+ 
+This dependency is a maven dependency that copies our dependencies and puts them into a jar file. A jar file is a file that runs our Java application. We'll explain
+more about jar files in a little bit. Make sure this dependency is added under:
+ ``` xml
+ <pluginManagement>
+<!-- Insert Dependency here -->
+</pluginManagement
+```
+
+
+Next, we have to create a Procfile. This is a text file that that configures Heroku to set up your Java application. Our Procfile must look like:
+
+```ignorelang
+web: java $JAVA_OPTS -cp target/classes:target/dependency/* <project main package>
+```
+
+In our case, if you download our catch-up code, our Procfile looks like:
+```ignorelang
+web: java $JAVA_OPTS -jar service/target/service-1.0-SNAPSHOT-jar-with-dependencies.jar
+```
+
+This Procfile specifies that our application will use the web, indicated by the "web" portion, and our "java" section specifies how to start it up. 
+The last portion of the Procfile creates our jar file. A jar file is a file used to run a Java application. You can use a jar file to run your Java code anywhere. It's easily exported and
+is how Heroku runs our application. Our Procfile is located in our project's root directory.
+
+The next configuration we need is a <strong> system.properties </strong> file. This is a file that Heroku uses to define our application's properties, such as the Java and Maven versions. This properties
+file is different from our <strong> service/.../resources/properties </strong> directory as this property file is only for Heroku.
+
+Our system.properties looks like
+```ignorelang
+java.runtime.version=11
+maven.version=3.6.2
+```
+Make sure this file is in our root directory so that Heroku can easily identify this file.
+
+Once we have all of our configurations set up, we can begin deploying our backend. Navigate to our backend directory and run the command:
+
+```shell
+mvn clean install
+```
+
+This will clean our application and build it again.
+Once we verified that our application successfully built, we follow the same steps as deploying our frontend code.
+
+If we have not committed our changes to the master branch make so to do so with:
```

Comment:
Maybe add another comment down here on how to merge changes on their current branch to master